src/order/compactly_generated.lean

```diff
@@ -470,6 +473,21 @@ theorem is_complemented_of_Sup_atoms_eq_top (h : Sup {a : α | is_atom a} = ⊤)
         refine hc2.directed_on.mono (λ s t, Sup_le_Sup) } },
     { rcases set.mem_sUnion.1 ha with ⟨s, sc, as⟩,
       exact (hc1 sc).2.2 a as } }
+end
+
+lemma exists_set_independent_of_Sup_atoms_eq_top (h : Sup {a : α | is_atom a} = ⊤) :
+  ∃ s : set α, complete_lattice.set_independent s ∧ Sup s = ⊤ ∧ ∀ a ∈ s, is_atom a :=
+begin
+  obtain ⟨s, s_ind, s_top, s_atoms⟩ := exists_set_independent_is_compl_Sup_atoms h (⊥ : α),
+  refine ⟨s, s_ind, eq_top_of_is_compl_bot (is_compl.symm s_top), s_atoms⟩,
+end
+
+/-- See Theorem 6.6, Călugăreanu -/
```

Comment:
This docstring doesn't help a reader. It's OK to give a reference as an addition to a useful docstring but IMHO docstrings should be readable and useful without reading bibliographic references. BTW, we have a `bib` file and a standard format for bibliographic references.

Reply:
This docstring isn't new. Winston didn't change it.